package.json

```diff
@@ -84,5 +84,9 @@
     "theme",
     "globals.d.ts",
     "index.d.ts"
-  ]
+  ],
+  "dependencies": {
+    "@emotion/styled": "^11.9.3",
+    "react-router-dom": "^6.3.0"
+  }
 }
```

Comment:
so I know we talked about this on discord, but I'd be concerned about having these as actual dependencies, rather than peer dependencies as `izakaya` uses these packages as well....

could you work with @zeulb or @aw2934 to confirm that this 100% works in `izakaya`?

Reply:
@smurmus i'll remove these, they shouldn't have popped into the PR